propagation/onboarding.md

```diff
@@ -0,0 +1,76 @@
+# Algorithm Onboarding Service
+
+APEx offers an algorithm onboarding service as part of its propagation services to ensure that project results remain
+accessible to the EO community beyond the project's completion. This service aims to sustain and promote the use of
```

Comment:
algorithm propagation is dedciated to operationalise workflows and algorithms, we need to leep that separate from other project results..

Reply:
I have updated the text to 'project workflows and algorithms'